snakeviz/cli.py

```diff
@@ -108,9 +109,15 @@ def main(argv=None):
         parser.error('invalid port number %d: use a port between 0 and 65535'
                      % port)
 
+    # Before starting tornado set the eventloop policy for windows and python 3.8 compatibility
+    # https://github.com/tornadoweb/tornado/issues/2608
+    if sys.platform == 'win32':
+        asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
```

Comment:
Perhaps worth copying https://github.com/python/pyperformance/pull/65/files here?

Reply:
Perfect, thanks for the tip!